### PR TITLE
chore(RHTAPWATCH-539): Run yamllint checks for each opened PR

### PR DESCRIPTION
**Pull request by Kousalya1998**
- Add yaml lint in the makefile
- Add yaml lint to tekton 

Ticket:
[RHTAPWATCH-539](https://issues.redhat.com/browse/RHTAPWATCH-539)

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+yamllint = "1.28.0"
```
Comment (yftacherzog) [inline Pipfile:9]: add newline

**Comment by yftacherzog** [comment]
nice work!

**Comment by avi-biton** [comment]
LGTM, other than the new line in `Pipfile`

File: config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ roleRef:
   name: read-grafana
   apiGroup: rbac.authorization.k8s.io
 ---
-# Run a cronjob to every 10 minutes check if the "prometheus-appstudio-ds" datasource 
+# Run a cronjob to every 10 minutes check if the "prometheus-appstudio-ds" datasource
```
Comment (Omeramsc) [inline config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml:33]: Is this relevant for this patch?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -15,6 +15,7 @@ This section describes the main steps required before you can add your code and
 * Basic tools: `curl`, `tar`, `make``.
 * Some container engine: either `docker` or `podman`.
 * go: v1.20
+* Python3 and pipenv.
```
[the remark this answers is not in the document]
Comment (Kousalya1998) [reply CONTRIBUTING.md:18]: Will check and update the version

File: config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml (hunk 2 above)
Comment (Omeramsc) [inline config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml:33]: I'll recommend adding YAML fixes to a separate commit (same PR is ok)

File: test/promql/tests/data_plane/component_errors_test.yaml [hunk 4]
```diff
@@ -77,8 +77,9 @@ tests:
       # HAS experienced ~10% (10.83%) deletion failure rate with counter reset, should still alert
       - series: 'has_component_deletion_total{namespace="application-service", pod="has", source_cluster="cluster01"}'
         values: '10+10x64'
+      # Increment failures by 1 for 20min, reset counter to 0 and increment failures by 10 for another 5 mins
       - series: 'has_component_failed_deletion_total{namespace="application-service", pod="has", source_cluster="cluster01"}'
-        values: '1+1x20 0+10x5 50x38' # increment failures by 1 for 20min, reset counter to 0 and increment failures by 10 for another 5 mins
+        values: '1+1x20 0+10x5 50x38'
```
Comment (Omeramsc) [inline test/promql/tests/data_plane/component_errors_test.yaml:82]: why was the comment moved? is it relevant for this PR?

File: config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml (hunk 2 above)
Comment (Kousalya1998) [reply config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml:33]: guess this time its okay..

File: test/promql/tests/data_plane/component_errors_test.yaml (hunk 4 above)
Comment (Kousalya1998) [reply test/promql/tests/data_plane/component_errors_test.yaml:82]: Yes, it has some issue with that comment when I ran yamllint. so I had to move it either above or below.

**Review by hmariset — APPROVED** [review]
LGTM

**Comment by yftacherzog** [comment]
@Kousalya1998 it seems as you had some open discussions with @Omeramsc. The thing to do, next time, before merging is to get the confirmation from such reviewers that it's ok to proceed.

**Comment by Kousalya1998** [comment]
> @Kousalya1998 it seems as you had some open discussions with @Omeramsc. The thing to do, next time, before merging is to get the confirmation from such reviewers that it's ok to proceed.

@yftacherzog Thanks for letting me know. I closed it because I saw LGTM from Homaja and Avi. Sure, will wait until I get LGTM from all the reviewers and then merge.

**Comment by yftacherzog** [comment]
> > @Kousalya1998 it seems as you had some open discussions with @Omeramsc. The thing to do, next time, before merging is to get the confirmation from such reviewers that it's ok to proceed.
> 
> @yftacherzog Thanks for letting me know. I closed it because I saw LGTM from Homaja and Avi. Sure, will wait until I get LGTM from all the reviewers and then merge. 

We don't have to get lgtm from all reviewers but if someone already raised some issues then those need to be confirmed to be addressed.

**Pull request merged**